src/bindings/python/tests/test_runtime/test_sync_infer_request.py

```diff
@@ -36,7 +35,8 @@ def create_model_with_memory(input_shape, data_type):
     add = ops.add(rv, input_data, name="MemoryAdd")
     node = ops.assign(add, "var_id_667")
     res = ops.result(add, "res")
-    model = Model(results=[res], sinks=[node], parameters=[input_data], name="name")
+    model = Model(results=[res], sinks=[node],
+                  parameters=[input_data], name="name")
```

Comment:
Could you please also remove style changes like this in this file?

Reply:
Hi @akuporos , 
     Hope you are well. I just made the changes you requested . Please check whether everything is ok as per your request . Please comment if any changes are required.

Thanks Bhaskar